sql/core/src/main/scala/org/apache/spark/sql/DataFrameReader.scala

```diff
@@ -727,6 +727,27 @@ class DataFrameReader private[sql](sparkSession: SparkSession) extends Logging {
    * a record can have.</li>
    * <li>`maxCharsPerColumn` (default `-1`): defines the maximum number of characters allowed
    * for any given value being read. By default, it is -1 meaning unlimited length</li>
+   * <li>`unescapedQuoteHandling` (default `STOP_AT_DELIMITER`): defines how the CsvParser
```

Comment:
Seems fine. Can you also update `DataStreamReader.scala`, `readwriter.py`, and `streaming.py`?

Reply:
Address 1770c56 fix this

Reply:
Address 84c1d59 try to fix python file,  I'm not familiar with Python :(

Reply:
@HyukjinKwon also needs to be update `readwriter.pyi` , and `streaming.pyi`? Is that right?